### ISSUE TITLE
Internal exception occurred while opening

### ISSUE THREAD

**Issue by J-Mendes** [plaid/plaid-link-android#74]
Hi,
I've detected that when I launch the SDK on a device with API 24 it just shows the initial loading for a moment and then exits with an internal exception. Below is the actual error that I'm getting when this happens. This occurs in the version 1.4.1.
```
PlaidError(
    displayMessage=Internal exception occurred,
    errorCode=499,
    errorMessage=[
        com.plaid.o3$d.accept(SourceFile:4),
        io.reactivex.internal.observers.LambdaObserver.onError(LambdaObserver.java:77),
        com.uber.autodispose.HalfSerializer.onError(HalfSerializer.java:148),
        com.uber.autodispose.AutoDisposingObserverImpl.onError(AutoDisposingObserverImpl.java:98),
        io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.checkTerminated(ObservableObserveOn.java:281),
        io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:172),
        io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:255),
        io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:124),
        android.os.Handler.handleCallback(Handler.java:751),
        android.os.Handler.dispatchMessage(Handler.java:95),
        android.os.Looper.loop(Looper.java:154),
        android.app.ActivityThread.main(ActivityThread.java:6077),
        java.lang.reflect.Method.invoke(Native Method),
        com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:866),
        com.android.internal.os.ZygoteInit.main(ZygoteInit.java:756)],
    errorType=null cannot be cast to non-null type android.content.pm.PackageInfo,
    linkExitMetadata=LinkExitMetadata(linkSessionId=null, institutionName=null, institutionId=null, requestId=null, status=null))
```
Thanks

**Comment by amytang0** [comment]
Hi J-Mendes, thanks for the bug report, we are fixing this in 2.0, releasing roughly around June 15. Do you mind sharing what device this is occurring on? Thanks!

**Comment by Varun7691** [comment]
Is this fixed in 2.0.0-rcx ?!

**Comment by amytang0** [comment]
Yes, this is fixed in 2.0.0-rc3

**Comment by J-Mendes** [comment]
Hi @amytang0 it's occuring on a Nexus 5X and on a Samsung A3

**Comment by J-Mendes** [comment]
I just confirmed that this is fixed on release 2.0.0

**Comment by JeroenMols** [comment]
Thanks for letting us know!
